### PR TITLE
update docs on streaming option enum values in pqs scribe

### PR DESCRIPTION
**Pull request by dasormeter**
Add docs on the pqs scribe params  `--pipeline-ledger-stop` and  `--pipeline-datasource`

### CODE REVIEW

File: docs/2.7.9/docs/query/pqs-user-guide.rst [hunk 1]
```diff
@@ -527,10 +533,21 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
...
+
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream and related reassignments for a set of parties. For transactions it lists only creates and archives, but no other events. Omits all events on transient contracts, i.e., contracts that were both created and archived in the same transaction.
```
Comment (cathyjung-da) [inline docs/2.7.9/docs/query/pqs-user-guide.rst:549]: "i.e." needs to be replaced with "that is"

File: docs/2.9.0/docs/query/pqs-user-guide.rst [hunk 2]
```diff
@@ -531,10 +537,21 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
...
+
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream and related reassignments for a set of parties. For transactions it lists only creates and archives, but no other events. Omits all events on transient contracts, i.e., contracts that were both created and archived in the same transaction.
```
Comment (cathyjung-da) [inline docs/2.9.0/docs/query/pqs-user-guide.rst:553]: "i.e." needs to be replaced with "that is"

**Review by cathyjung-da — DISMISSED** [review]
"i.e." needs to be replaced with "that is"

File: docs/2.9.0/docs/query/pqs-user-guide.rst (hunk 2 above)
Comment (dasormeter) [reply docs/2.9.0/docs/query/pqs-user-guide.rst]: reworded

File: docs/2.7.9/docs/query/pqs-user-guide.rst [hunk 3]
```diff
@@ -527,10 +533,21 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
...
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
+-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises and transient contracts, but excludes interface views.
```
Comment (carrielaben-da) [inline docs/2.7.9/docs/query/pqs-user-guide.rst:550]:
```suggestion
-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives, and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises, and transient contracts, but excludes interface views.
```

File: docs/2.8.4/docs/query/pqs-user-guide.rst [hunk 4]
```diff
@@ -528,10 +534,21 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
...
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
+-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises and transient contracts, but excludes interface views.
```
Comment (carrielaben-da) [inline docs/2.8.4/docs/query/pqs-user-guide.rst:551]:
```suggestion
-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives, and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises, and transient contracts, but excludes interface views.
```

File: docs/2.9.0/docs/query/pqs-user-guide.rst [hunk 5]
```diff
@@ -531,10 +537,21 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
...
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
+-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises and transient contracts, but excludes interface views.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/query/pqs-user-guide.rst:554]:
```suggestion
-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives, and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises, and transient contracts, but excludes interface views.
```

File: docs/3.1/docs/query/pqs-user-guide.rst [hunk 6]
```diff
@@ -531,10 +537,22 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
...
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
+-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises and transient contracts, but excludes interface views.
```
Comment (carrielaben-da) [inline docs/3.1/docs/query/pqs-user-guide.rst:554]:
```suggestion
-  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives, and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
-  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises, and transient contracts, but excludes interface views.
```

**Review by carrielaben-da — APPROVED** [review]
Commas

**Pull request merged**